auth-login.js

```
@@ -3,302 +3,303 @@
 // @author       Lynn
 // @match        https://tysfrz.ccut.edu.cn/portal/login.html*
 // @grant        none
 // ==/UserScript==
 
-
 (function() {
     'use strict';
 
     // 添加样式
     const style = document.createElement('style');
     style.textContent = `
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)